app/jobs/last_active_job.rb

```diff
@@ -15,6 +15,6 @@ def perform(user_id, time_last_active)
 
     time_last_active = Time.zone.at(time_last_active)
     user.update_columns(last_active_at: time_last_active) # rubocop:disable Rails/SkipsModelValidations
-    User.update_index("stafftools#user") { user }
+    User.update_index("user#user") { user }
```

Comment:
Can you explain why the argument here is `user#user`

Reply:
Since we deleted the `StafftoolsIndex` we no longer update that index. The user type is now defined on the `UserIndex`. We do the same thing here: 
https://github.com/education/classroom/pull/1588/files#diff-4676c008b11a5480d73d4a6de01e45b9

Reply:
So is this syntax a Chewy thing to
Maintain compatibility? If there’s only one type per index why would we need to specifu the index and the type?

Reply:
> So is this syntax a Chewy thing to maintain compatibility?
It must be.